.gitignore

```diff
@@ -24,6 +24,27 @@ haters/
 .idea/**/uiDesigner.xml
 .idea/**/dbnavigator.xml
 
+# Eclipse
+# -------
+*.pydevproject
```

Comment:
Are we going to have Python/C here?

Reply:
Probably not, I didn't edit it, it was pulled straight from another repo.
I'll tiddy it up.